README.md

```diff
@@ -1 +1,170 @@
+  - `self.find(id)` - returns an instance of `Product` where the value of the id field in the CSV matches the passed parameter
+
+#### CSV Data File
+The data for the products CSV file consists of:
+
+| Field          | Type    | Description
+|----------------|---------|------------
+| ID             | Integer | A unique identifier for that Product
+| Description | String | The product description
+| Price      | Float  | The product's price
+| Quantity     | Integer  | The product's quantity
+
+To create the relationship between the orders and the products use the `order_products` CSV file. When you use this file, you can ignore the product data in the existing order CSV file. The data for this file, in order in the CSV, consists of:
+
+| Field      | Type    | Description
+|------------|---------|------------
+| Order ID | Integer | A unique identifier corresponding to an Order
+| Product ID   | Integer | A unique identifier corresponding to an Product
+
+This type of table, where records from other tables are associated with each other, is often called a _join table_. We'll talk about them as a class in a few weeks.
+
+## Wave 3
+### Learning Goals
+- Use inheritance to share some behavior across classes
+- Enhance functionality built in Wave 1
+- Add tests for all new classes and inherited functionality
+
+### Requirements
+
+For wave 3, you will create two new classes: `Customer` and `OnlineOrder`.
+
+ The `OnlineOrder` class will inherit behavior from the `Order` class and include additional data to track the customer and order status. An **instance** of the `Customer` class will be used _within_ each **instance** of the `OnlineOrder` class.
+
+Each class should get its own file under the `lib/` directory, and each already has a spec file with stub tests.
+
+#### Customer
+Create a `Customer` class within the `Grocery` module.
+
+Each new Customer should include the following attributes:
+- ID
+- email address
+- delivery address information
+
+#### CSV Data File
+The data for the products CSV file consists of:
+
+| Field          | Type    | Description
+|----------------|---------|------------
+| Customer ID | Integer | A unique identifier corresponding to the Customer
+| Email   | String | The customer's e-mail address
+| Address 1 | String | The customer's street address
+| City |  String | The customer's city
+| State | String | The customer's state
+| Zip Code | String | The customer's zip code
+
+
+#### OnlineOrder
+Create an `OnlineOrder` class which will inherit behavior from the `Order` class.
+
+Each new OnlineOrder should include the following attributes:
+- A customer object
+- A fulfillment status
+  - pending, paid, processing, shipped or complete
+
+It should include the following updated functionality:
+- The `total` method should be the same, except it will add a $10 shipping fee
+- The `add_product` method should be updated to permit a new product to be added if the status is either pending or paid
```

Comment:
I do want it to be permit - but I should make it more clear that it is *only* for these statuses